pyproject.toml

```
@@ -85,7 +86,7 @@ extend_exclude = ".tox,docs,generated,src/codegen/metadata,src/codegen/templates
 [tool.pytest.ini_options]
 addopts = "--doctest-modules --strict-markers"
 filterwarnings = ["always::ImportWarning", "always::ResourceWarning"]
-testpaths = [ "tests" , "generated/nidaqmx/_install_daqmx.py"]
+testpaths = [ "tests" , "generated/nidaqmx"]
```

Please don't leave extra spaces.
```suggestion
testpaths = ["tests" , "generated/nidaqmx"]
```